### PR TITLE
Zijie lint fix for reports component Phase 10

### PR DESCRIPTION
**Pull request by iTvX**
# Description
Lint fix for Reports component, this is phase 10.

## Files changed
All js/jsx file in src/components/Reports/ProjectReport/

eslint test:
CleanShot 2023-10-04 at 00 46 05@2x

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint "src/components/Reports/ProjectReport/*.js" "src/components/Reports/ProjectReport/*.jsx"`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!

### CODE REVIEW

**Review by emunkh404 — APPROVED** [review]
Reviewed

**Comment by emunkh404** [comment]
I've reviewed your "Zijie lint fix" branch for the "reports component" in Phase 10, PR #1377. Most functions work well, but the "Total team report" is stuck loading. Overall, it's an improvement, but that issue needs attention

image

**Review by DavidC0126 — APPROVED** [review]
Hi @iTvX I tested the PR and it works as intended! I also noticed that the chart loading speed is faster! Good Job.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/74082492/31f05864-22d2-4349-ac2d-bec9d50a3f92

**Comment by Iris-Song** [comment]
I have tested the PR and it works as intended! 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58033867/286bf644-bf29-47d4-84da-8b14e1718a4d

**Comment by Iris-Song** [comment]
I have tested the PR and it works as intended! 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58033867/286bf644-bf29-47d4-84da-8b14e1718a4d

**Comment by Iris-Song** [comment]
I have tested the PR and it works as intended! 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58033867/286bf644-bf29-47d4-84da-8b14e1718a4d

**Review by lacnoskillz — DISMISSED** [review]
After pulling your changes and testing them locally and comparing it to development it appears it functions the same. Loading times seem to be the same on my end. (the show total team report does take a bit longerthan the others but i assume its because it has alot more data to load). 
I checked :
 Show/hide total project report button/functionality
  Show/hide total people report button/functionality
 Show total/hide team report button/functionality
 And also the categorys:
 Projects
 People
 Teams
 and tested changing start dates and filters like Inactive/Active/All
 Which all seemed to work
 

Screenshot (211)
Screenshot (210)
Screenshot (209)
Nice work!

**Review by Jiyuanxia — DISMISSED** [review]
I have tested your pull request and it works as intended. Well done!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92963742/656181a0-bee3-4ad3-a11e-ab7a9af21494

**Comment by palakgosalia** [comment]
Hi, I have tested your changes and it looks good.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104417272/a91f77c7-efc5-4607-b89d-025a8a5ea863

**Review by savio-henrique — COMMENTED** [review]
Hi @iTvX , I've tested your PR, as the video below shows, there were no issues with the main functionalities, although the total teams report was loading so slow that it taked about a minute to show up! (I don't know if it is related to this PR or not)

As you said, the linting of the source code passed all tests in ESLint, but I've found an error while testing in the main reports page, both demonstrated by the screenshots below.

---

## Video and Screenshots

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/83360232/c82d626c-32cc-449b-b9e1-17cb6161fe1f

terminal

error

---

Nice work on this PR! :+1:

**Review by yi-lin-1234 — APPROVED** [review]
Hello, functionality of the ProjectReport pages is the same as before. And, the eslint fix worked as expected. Great work!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92185695/74ffc842-1f81-4b54-84ed-3aea14e9831a

Screenshot 2023-10-25 at 10 43 58 AM

**Review by Chuehleo — APPROVED** [review]
Everything works as expected, great job!

Uploading HGN APP - Google Chrome 2023-11-03 00-34-22.mp4…

**Review by malikjahanzaib — APPROVED** [review]
Hi @iTvX, no lint errors and app works normally. Nice job!!!

Test screenshot/rec:

image


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46221334/b515a541-c279-4cc2-a0b8-bc648feea874

File: src/components/Reports/ProjectReport/index.js [hunk 1]
```diff
@@ -1 +1,2 @@
-export { ProjectReport } from './ProjectReport';
+// eslint-disable-next-line import/prefer-default-export
```
Comment (EvianTan) [inline src/components/Reports/ProjectReport/index.js:1]: Please fix it instead of disabling it.

Comment (iTvX) [reply src/components/Reports/ProjectReport/index.js]: fixed

**Comment by EvianTan** [comment]
image
Failed to build and cannot login via incognito window, Same as this [comment](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1356/files#r1391983106)

**Pull request closed without merging**